stubs/slumber/slumber/__init__.pyi

```diff
@@ -1,30 +1,39 @@
+from _typeshed import Self
 from typing import Any
 
+from requests import Response, Session
+from requests.sessions import _Auth, _Data, _Files
+
+from .serialize import Serializer
+
+__all__ = ["Resource", "API"]
+
 class ResourceAttributesMixin:
-    def __getattr__(self, item): ...
+    # Exists at runtime:
+    def __getattr__(self, item: str) -> Any: ...
 
 class Resource(ResourceAttributesMixin):
-    def __init__(self, *args, **kwargs) -> None: ...
-    def __call__(self, id: Any | None = ..., format: Any | None = ..., url_override: Any | None = ...): ...
-    def as_raw(self): ...
-    def get(self, **kwargs): ...
-    def options(self, **kwargs): ...
-    def head(self, **kwargs): ...
-    def post(self, data: Any | None = ..., files: Any | None = ..., **kwargs): ...
-    def patch(self, data: Any | None = ..., files: Any | None = ..., **kwargs): ...
-    def put(self, data: Any | None = ..., files: Any | None = ..., **kwargs): ...
-    def delete(self, **kwargs): ...
-    def url(self): ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
```

Comment:
Since `args` is ignored:

```suggestion
    def __init__(self, *args: object, **kwargs: Any) -> None: ...
```

Reply:
This is intentional. This is designed to be subclassed, as far as I understand. And it can be used in subclasses. 

It is easier to redefine `*Any, **Any` than `*object, **Any` from my perspective.